app/collins/models/conversions.scala

```diff
@@ -136,7 +137,10 @@ object conversions {
       (json \ "UPDATED").asOpt[Timestamp],
       (json \ "DELETED").asOpt[Timestamp],
       (json \ "ID").as[Long],
-      (json \ "STATE").asOpt[State].map(_.id).getOrElse(0)))
+      (json \ "STATE") match {
+        case JsNull => State.ANY_STATUS
```

Comment:
If you say so.  I would think State and Status should always be well defined, but reality and (my) theory often disagree.